src/utils/find.ts

```diff
@@ -28,26 +33,43 @@ function matches(node: VNode, selector: FindAllComponentsSelector): boolean {
   return false
 }
 
+function notNullOrUndefinedOrVoid<Node>(
+  value: Node | null | undefined | void
+): value is Node {
+  return !!value
+}
+
 /**
  * Collect all children
  * @param nodes
  * @param children
  */
-function aggregateChildren(nodes, children) {
+function aggregateChildren(nodes: VNode[], children: VNodeNormalizedChildren) {
   if (children && Array.isArray(children)) {
-    ;[...children].reverse().forEach((n: VNode) => {
-      nodes.unshift(n)
-    })
+    const reversedChildren = [...children]
+      .reverse()
+      .filter(notNullOrUndefinedOrVoid)
+    console.log(reversedChildren)
+    reversedChildren.forEach(
+      (node: string | number | boolean | VNodeArrayChildren | VNode) => {
+        nodes.unshift(node)
```

Comment:
I need help here, please.
If I follow correctly until that point, `children` is of type `VNodeNormalizedChildren`.
That means its elements, after filtering the potential null/undefined ones (which I added to be safe), can be either `string | number | boolean | VNodeArrayChildren | VNode` (see https://github.com/vuejs/vue-next/blob/24168bbb333727417ddcb310c615afc55ee923ad/packages/runtime-core/src/vnode.ts#L85-L92) . The current code considers it is always a VNode. Can we be sure of that? 
If yes, I cast force the cast and add a comment explaining why.
If no, we need to handle the other cases, but I'm not sure what needs to be done.
cc @dobromir-hristov

Reply:
I think I was just collecting all children, then in the `match` I am filtering away the unnecessary. 
Children can be other things, but I did not yet find such cases cases. 

`String` means its just a plain DOM element. 
`VNode` is Vue component instance.
Boolean I have no idea when 😆 
`VNodeArrayChildren` is probably for multi root components? idk.. :/

Reply:
Right, I did not manage to reproduce a case where we have `VNodeArrayChildren` either. To be safe, I added a branch if we end up with it, and call aggregateChildren on them. We should be safe in all cases.